zipline/data/minute_bars.py

```diff
@@ -549,7 +549,9 @@ def last_date_in_output_for_sid(self, sid):
         with open(sizes_path, mode='r') as f:
             sizes = f.read()
         data = json.loads(sizes)
-        num_days = data['shape'][0] / self._minutes_per_day
+        # use integer division so that the result is an int for pandas index later
+        # https://github.com/pandas-dev/pandas/blob/master/pandas/tseries/base.py#L247
```

Comment:
Looks like the flake8 tests failed because line 552 is too long (and probably 553 as well)